Libs/RaptureCore/src/integrationtest/java/rapture/kernel/IndexApiImplTest.java

```diff
@@ -0,0 +1,288 @@
+package rapture.kernel;
```

Comment:
That's what it's supposed to be.
ORAC:Rapture qqqq$ ls -l Libs/RaptureCore/src/
total 0
drwxr-xr-x  5 qqqq  staff  170 Oct  3 09:35 integrationTest
drwxr-xr-x  5 qqqq  staff  170 Oct  3 09:35 main
drwxr-xr-x  4 qqqq  staff  136 Oct  3 09:35 test

ORAC:Rapture qqqq$ ls -l Libs/RaptureCore/src/integrationTest/java/rapture/kernel/IndexApiImplTest.java
-rw-r--r--  1 qqqq  staff  14570 Nov 11 19:22 Libs/RaptureCore/src/integrationTest/java/rapture/kernel/IndexApiImplTest.java